web/frontend/src/components/board/index.svelte

```diff
@@ -95,7 +95,7 @@ const writeAnalog = async () => {
 
 const readAnalog = async () => {
   try {
-    const value = await boardClient.readAnalogReader(analogPinName);
+    const {value} = await boardClient.readAnalogReader(analogPinName);
```

Comment:
i'm pretty sure the lint error is because rdk doesn't point to the new typescript changes yet so it still thinks the return value is a number? im waiting to merge this until next TS release. But can still change to this syntax if preferred.

Reply:
Ah that's certainly the case as well, I think you'll start getting this error once the TS release is resolved. Either way, this spacing looks a little nicer to me. Thanks!